tempest/api/workloadmgr/security_groups/test_security_group_default.py

```
@@ -0,0 +1,322 @@
+            reporting.add_test_step(
+                "Create full snapshot", tvaultconf.FAIL
+            )
+            raise Exception("Snapshot creation failed")
+        LOG.debug("\nFull snapshot ids : {}\n".format(snapshot_id))
+        return snapshot_id
+
+    def _restore_through_wlm_cli(self, snapshot_id):
+        restore_command = (
+                command_argument_string.restore_security_groups + " " + snapshot_id
+        )
+        rc = cli_parser.cli_returncode(restore_command)
+        if rc != 0:
+            reporting.add_test_step(
+                "Execute restore security groups command", tvaultconf.FAIL
+            )
+            raise Exception("Command did not execute correctly")
+        else:
+            reporting.add_test_step(
+                "Execute restore security groups command", tvaultconf.PASS
+            )
+            LOG.debug("Command executed correctly")
+
+        time.sleep(10)
+        # DB verification of the command execution
+        wc = query_data.get_snapshot_restore_status(
+            tvaultconf.security_group_restore_name, snapshot_id
+        )
+        LOG.debug("Security group restore status: " + str(wc))
+        while str(wc) != "available" or str(wc) != "error":
+            time.sleep(5)
+            wc = query_data.get_snapshot_restore_status(
+                tvaultconf.security_group_restore_name, snapshot_id
+            )
+            LOG.debug("Snapshot restore status: " + str(wc))
+            if str(wc) == "available":
+                reporting.add_test_step(
+                    "Security group restore command verification",
+                    tvaultconf.PASS,
+                )
+                self.created = True
+                break
+            else:
+                if str(wc) == "error":
+                    reporting.add_test_step(
+                        "security group restore command verification",
+                        tvaultconf.FAIL,
+                    )
```

test script status also need to be marked as FAIL explicitly

---

done